### ISSUE TITLE
external link removed

### ISSUE THREAD

**Issue by davidnuescheler** [adobe/helix-pipeline#353]
it looks like the resolution of #253 removes external links in general...

so it turns:

```
Example: [https://helix-home-adobe.project-helix.page/README.html](https://helix-home-adobe.project-helix.page/README.html)
```

into

```

<p>Example: <a>https://helix-home-adobe.project-helix.page/README.html</a></p>

```

generally, i think the entire threat model behind #253 is invalid in my mind as discussed at the last hackathon, so this may open a bigger discussion. maybe something we should pick up again @lkrapf @asanso @ramboz @trieloff ?

**Comment by asanso** [comment]
> i think the entire threat model behind #253 is invalid in my mind as discussed at the last 
@davidnuescheler  i agree.
See also 

>The initial plan was to act as a malicious author and escape the XSS sandbox as done in
>https://github.com/adobe/helix-pipeline/issues/253 .
>The problem with that approach was that in the meantime Helix project added the possibility to
>directly add HTML and Javascript making this bug obsolete.

in the [security summary](https://github.com/adobe/helix-home/blob/master/hackathons/5-bsl-security.pdf)

**Comment by ramboz** [comment]
@davidnuescheler I wonder which cli version you are using. As far as I can see, the latest 4.0.6 doesn't yet have pipeline 2.2.0 which contains the changes. So if links are broken in that one, it may be coming from a separate issue. Also, in my tests, I had a page with several images working fine on the PR

**Comment by ramboz** [comment]
Filed https://github.com/adobe/htlengine/issues/66 to track this and https://github.com/adobe/htlengine/pull/67 to fix it.

**Comment by tripodsan** [comment]
it’s written in the breaking changes notes: you need to use ```
@context = “unsafe”
```

since the DOM is already purified in the pipeline.

**Comment by tripodsan** [comment]
see https://github.com/adobe/project-helix.io/blob/8e11173d68957b16ec876c408903b71c05918540/src/html.htl#L56

**Comment by tripodsan** [comment]
but this will all not be required anymore once we implement the direct dom injection in the htlegine.

**Comment by davidnuescheler** [comment]
@ramboz i was using 4.0.6

**Comment by tripodsan** [comment]
this should be fixed with #337
